vacation_calendar/templates/dashboard.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dashboard</title>
+    <link rel="stylesheet" href="style.css" />
```

Comment:
I would use Django's static files app to make sure this line will work in all environments. In production, you typically don't serve assets (static files) from the same server you're hosting your webapp on.

This is another way of saying, in production, your Django app will likely not have this file on disk ready to be served at this relative path, unless you use `whitenoise` or a similar solution.